containers/server-image/Dockerfile

```diff
@@ -83,3 +88,34 @@ LABEL org.opensuse.reference="${REFERENCE_PREFIX}/server:%PKG_VERSION%.%RELEASE%
 
 CMD ["/usr/lib/systemd/systemd"]
 HEALTHCHECK --interval=5s --timeout=5s --retries=5 CMD ["/usr/bin/systemctl", "is-active", "multi-user.target"]
+
+LABEL MIGRATE="/usr/bin/podman run -it --rm     -v cgroup:/sys/fs/cgroup:rw \
```

Comment:
I don't remember, why we need cgroups volume?

Reply:
Actually I can't remember (not sure if @cbosdo can), but it's present also when we run container from scratch. I can test if it's really required.

Reply:
IIRC this is needed to run systemd inside the container. Not sure if this is useful in the label.